web-access-control/web-access-control-test-manifest.ttl

```diff
@@ -26,31 +26,31 @@ manifest:protected-operation-not-read-resource-default-AWC
 manifest:protected-operation-read-resource-access-R
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#access-modes> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/protected-operation/read-resource-access-R.feature> .
```

Comment:
Do we want to test for non-modification in addition to `403`?

Reply:
@kjetilk Can you look at this please?

Reply:
Yes, we should do that as a general practice.